.github/workflows/cluster-rotate.yml

```diff
@@ -177,8 +180,10 @@ jobs:
             esac
           done
 
-      - name: Save new CLUSTER_ID
-        run: echo "${{ steps.set_cluster_id.outputs.cluster_id }}" > CLUSTER_ID
+      - name: Save new CLUSTER_ID and set node kubeconfig
+        run: |
+          echo "${{ steps.set_cluster_id.outputs.cluster_id }}" > CLUSTER_ID
+          cp ${KUBECONFIG} ~/.kube/config
```

Comment:
How is this change related to deleting namespaces?

Reply:
ops, this is something I was testing. Its just to set the kubeconfig on the node running the job. I can create another commit for it.